Tuto-GUDHI-persistent-entropy.ipynb

```diff
@@ -0,0 +1,823 @@
+{
```

Comment:
<p>We probably want to link to a more recent version of the doc, the link here doesn't seem to work.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GUDHI/TDA-tutorial/pull/52/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Right, I changed it by https://gudhi.inria.fr/python/latest/representations.html</p>

Reply:
<p>Though very minor, I think "refer the notebook Tuto-GUDHI-representation" could be linked, like "refer to <a href="https://github.com/GUDHI/TDA-tutorial/blob/master/Tuto-GUDHI-representations.ipynb" target="_blank">this notebook</a> ". </p>

Reply:
<p>I agree</p>